File: backend/src/main/java/me/performancereservation/domain/file/File.java
package me.performancereservation.domain.file;

import jakarta.persistence.*;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import me.performancereservation.domain.common.BaseEntity;

@Entity
@NoArgsConstructor
@Getter
public class File extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id; // 파일 ID

    @Column(name = "`key`")
    private String key; // Storage Key

    @Builder
    public File(Long id, String key) {
        this.id = id;
        this.key = key;
    }
}

File: backend/src/main/kotlin/me/performancereservation/domain/file/File.kt
package me.performancereservation.domain.file

import jakarta.persistence.*
import me.performancereservation.domain.common.BaseEntity

@Entity
class File (
    @field:GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    val id: Long? = null, // 파일 ID

    @Column(name = "`key`")
    val key: String // Storage Key
) : BaseEntity()
